sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -579,6 +579,59 @@ for (int i = 0; i < data.length; ++i) {
 }
 ```
 
+=== Appendix: Supported Parameter Combinations Per Hardware
+
+The tables below provide a list of the parameter combinations that
+`joint_matrix` implementations support on each supported vendors hardware type.
+
+==== Nvidia Tensor Cores Supported Combinations
+
+The complete set of matrix data types and shapes that are supported by the `ext_oneapi_cuda` backend are represented in the following table. Tm indicates the matrix element data type held by a "multiplicand" `joint_matrix`: i.e requiring `use::a` or `use::b`. Tc indicates the matrix element data type held by an "accumulator" `joint_matrix`: i.e requiring `use::accumulator`.
+
+IMPORTANT: When compiling for the `ext_oneapi_cuda` backend the target arch backend flag, `-Xsycl-target-backend --cuda-gpu-arch=sm_xx`, must be used, where `sm_xx` must be a Compute Capability that is equal to or greater than the appropriate Minimum Compute Capability. When an executable has been compiled for `sm_xx`, if the executable is run on a device with compute capability less than `sm_xx` then an error will be thrown. The mapping to Minimum Compute Capability from each supported parameter combination is specified in the following table.
+
+--
+[.center]
+|======================
+|Tm (`use::a` or `use::b`) |Tc (`use::accumulator`) |M |N |K | Minimum Compute Capability
+.3+|half  .3+|float
+|16 |16 |16| sm_70
+|8 |32 |16| sm_70
+|32 |8 |16| sm_70
+.3+|half  .3+|half
+|16 |16 |16| sm_70
+|8 |32 |16| sm_70
+|32 |8 |16| sm_70
+.3+|int8_t  .3+|int32_t
+|16 |16 |16| sm_72
+|8 |32 |16| sm_72
+|32 |8 |16| sm_72
+.3+|uint8_t  .3+|int32_t
+|16 |16 |16| sm_72
+|8 |32 |16| sm_72
+|32 |8 |16| sm_72
+|precision::tf32  |float |16 |16 |8| sm_80
+.3+|bfloat16  .3+|float
+|16 |16 |16 |sm_80
+|8 |32 |16 |sm_80
+|32 |8 |16 |sm_80
+|double  |double |8 |8 |4 |sm_80
+|======================
```

Comment:
Kind of a nit, but I think the table might look better if the last column also used "row span", similar to the first two columns.  For example, the first three rows in the table have "half" and "float" spanning all three rows.  Those three rows also all have "sm_70", so why not use "row span" there also?

Reply:
Thanks, yeah this is better. I've made this change.